dev/io.openliberty.persistence.3.2.thirdparty/resources/org/eclipse/persistence/exceptions/i18n/ValidationExceptionResource.nlsprops

```diff
@@ -639,6 +639,5 @@
 
 7359=No any ASM service available.
 
-7360=Database password was encrypted by deprecated algorithm.
-Reencrypt it by `passwordUpdate.sh` from eclipselink.zip bundle.
+7360=Database password was encrypted by deprecated algorithm. Reencrypt it by `passwordUpdate.sh` from eclipselink.zip bundle.
```

Comment:
A deprecated algorithm encrypted the database password.
Reencrypt the database password by running the `passwordUpdate.sh` file from the eclipselink.zip bundle.

Reply:
@is273 Under PR [28338](https://github.com/OpenLiberty/open-liberty/pull/28338), we just changed the message to the current version at request of @helyarp .  We would prefer to continue using the message as requested by @helyarp , since we have already requested this same message change back to the EclipseLink open source community where it is copied from.  All the changes we make here must be donated back to their source in EclipseLink, and since we have already submitted the prior change back to the source, we'd prefer not to change it yet again.

The only change we are trying to make here is to correct the error that the message was split across 2 lines, which turns it into 2 separate messages.  This is intended to be just one message.

Reply:
Thanks Tracy, Isaac.